wolfcrypt/src/rsa.c

```diff
@@ -4063,8 +4063,15 @@ int wc_MakeRsaKey(RsaKey* key, int size, long e, WC_RNG* rng)
         err = mp_mod(&key->d, &tmp1, &key->dP);
     if (err == MP_OKAY)                /* key->dQ = d mod(q-1) */
         err = mp_mod(&key->d, &tmp2, &key->dQ);
+#ifdef WOLFSSL_MP_INVMOD_CONSTANT_TIME
```

Comment:
Sean, what are the reasons for not making this on by default? Is there a performance impact?

Reply:
Yes, the exptmod version would be slower than a 'good' constant time modinv.